TowerForge/TowerForge/GameModule/GameWorld.swift

```diff
@@ -27,10 +28,18 @@ class GameWorld {
         gameEngine = GameEngine()
         gameMode = GameModeFactory.createGameMode(mode: mode, eventManager: gameEngine.eventManager)
         selectionNode = UnitSelectionNode()
+        powerUpSelectionNode = PowerUpSelectionNode(eventManager: gameEngine.eventManager)
+
+        grid = Grid(screenSize: screenSize)
+        if let scene = self.scene {
+            grid.generateTileMap(scene: scene)
+        }
```

Comment:
I think this section here is repeated.

Reply:
Removed, didn't catch this in the merge.